pages/components/avatar/avatar.html

```diff
@@ -0,0 +1,271 @@
+        <h1><a class="header-heading" href="../../../index.html">Arc UI</a></h1>
+      </div>
+      <div class="header__nav">
+        <nav>
+          <ul>
+            <li>
+              <a
+                target="_blank"
+                class="social-icons-wrapper"
+                href="https://twitter.com/devdeadviz"
+                ><i class="fab fa-twitter"></i
+              ></a>
+            </li>
+            <li>
+              <a
+                target="_blank"
+                class="social-icons-wrapper"
+                href="https://www.linkedin.com/in/devdeadviz/"
+              >
+                <i class="fab fa-linkedin"></i
+              ></a>
+            </li>
+            <li>
+              <a
+                target="_blank"
+                class="social-icons-wrapper"
+                href="https://github.com/devdeadviz"
+                ><i class="fab fa-github"></i
+              ></a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <div class="body-wrapper">
+      <aside class="drawer-wrapper">
+        <div class="drawer-header">
+          <h2 class="drawer-heading">Get Started</h2>
+        </div>
+        <hr />
+        <div class="drawer-options">
+          <a class="drawer-lists" href="../../get-started/intro.html">
+            <p class="drawer-list-content">Introduction</p>
+          </a>
+          <a class="drawer-lists" href="../../get-started/color.html">
+            <p class="drawer-list-content">Colors</p>
+          </a>
+          <a class="drawer-lists" href="../../get-started/typo.html">
+            <p class="drawer-list-content">Typography</p>
+          </a>
+        </div>
+        <div class="drawer-header">
+          <h2 class="drawer-heading">Components</h2>
+        </div>
+        <hr />
+        <div class="drawer-options">
+          <a class="drawer-lists" href="../avatar/avatar.html">
+            <p class="drawer-list-content">Avatars</p>
+          </a>
+          <a class="drawer-lists" href="../alert/alert.html">
+            <p class="drawer-list-content">Alert</p>
+          </a>
+          <a class="drawer-lists" href="../badge/badge.html">
+            <p class="drawer-list-content">Badges</p>
+          </a>
+          <a class="drawer-lists" href="../button/button.html">
+            <p class="drawer-list-content">Buttons</p>
+          </a>
+          <a class="drawer-lists" href="../card/card.html">
+            <p class="drawer-list-content">Cards</p>
+          </a>
+          <a class="drawer-lists" href="../image/image.html">
+            <p class="drawer-list-content">Image</p>
+          </a>
+          <a class="drawer-lists" href="../input/input.html">
+            <p class="drawer-list-content">Input</p>
+          </a>
+        </div>
+      </aside>
+      <main class="main-content m-5 width-90">
+        <h1 class="mb-4">Avatar</h1>
```

Comment:
MB4???

Reply:
The standard way of representing margins. Here mb-4 means margin-bottom: calc(4*var(--margin)).